src/gui/main_window.h

```diff
@@ -72,6 +72,7 @@ Q_OBJECT
 	/** Returns the application's localized name. */
 	QString appName() const;
 	
+	static int const EXIT_CODE_REBOOT;
```

Comment:
This must become constexpr. Even an inline constexpr function could do this perfectly.